File: src/test/java/ru/ezhov/rocket/action/caching/DiskCacheTest.java
package ru.ezhov.rocket.action.caching;

import org.junit.Test;

import java.io.File;
import java.net.MalformedURLException;
import java.net.URL;
import java.util.Optional;

public class DiskCacheTest {

    @Test
    public void test() throws MalformedURLException {
        DiskCache cache = new DiskCache();
        final Optional<File> fileOptional = cache.fromCache(new URL("https://www.elastic.co/favicon.ico"));
        if (fileOptional.isPresent()) {
            System.out.println(fileOptional);
        }
    }
}

File: cache/src/test/kotlin/ru/ezhov/rocket/action/cache/DiskCacheTest.kt
package ru.ezhov.rocket.action.cache

import org.junit.Test
import java.net.MalformedURLException
import java.net.URL

class DiskCacheTest {
    @Test
    @Throws(MalformedURLException::class)
    fun test() {
        val cache = DiskCache()
        cache.get(URL("https://www.elastic.co/favicon.ico"))?.let {
            println(it)
        }
    }
}